src/ansys/fluent/core/utils/event_loop.py

```diff
@@ -0,0 +1,26 @@
+"""Provides a module for event loop."""
+
+import asyncio
+from functools import partial
+
+loop = asyncio.get_event_loop()
+
+
+def execute_in_event_loop_threadsafe(f):
+    """Decorator to execute function in an event loop from another thread."""
```

Comment:
Start all descriptions with verbs ending in "s" or "es"

Reply:
Done. Thank you.